service/controller/chart/resource/status/create.go

```diff
@@ -13,6 +14,10 @@ import (
 	"github.com/giantswarm/chart-operator/service/controller/chart/key"
 )
 
+var (
+	blankDate = time.Date(0, time.January, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
For me this is a sign that the openapi rules in the app CRD are wrong.

If the helm release fails to deploy then there is never a date. What about fixing this in apiextensions instead?

Reply:
Good catch, That could be a more plausible option. I would take a look into apiextetions repo.